### PR TITLE
Update pacemakerd to use glib for command line handling

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: daemons/pacemakerd/Makefile.am [hunk 1]
```diff
@@ -33,4 +33,4 @@ endif
 pacemakerd_SOURCES	+= pcmkd_messages.c
 pacemakerd_SOURCES	+= pcmkd_subdaemons.c
 
-CLEANFILES = $(man8_MANS)
+CLEANFILES = $(man8_MANS) pacemaker.service pacemaker.combined.upstart pacemaker.upstart
```
Comment (kgaillot) [inline daemons/pacemakerd/Makefile.am:36]: These are currently removed with make distclean

I always get confused by the various cleans, but make clean is supposed to remove things created by make, while make distclean additionally removes things created by configure (which includes anything with a .in). So, I think leaving these for distclean is appropriate.

One of the reasons it's confusing is it's just a convention, I don't think there is any standard for how they are supposed to behave, but we should probably follow automake's conventions

There's also "mostlyclean" and "maintainer-clean", I try to ignore those whenever possible :)

**Review by kgaillot — COMMENTED** [review]
Looks great

File: daemons/pacemakerd/Makefile.am (hunk 1 above)
Comment (clumens) [reply daemons/pacemakerd/Makefile.am:36]: I've never used mostlyclean for anything.  My understanding of maintainer-clean is that it should essentially bring everything back to the state where you just checked out or unpacked the source.  I typically just use "git clean -fdx" for that, but maybe I should start using maintainer-clean so I don't drop files by accident.  If these are removed by distclean, I can just get rid of that patch.

Comment (clumens) [reply daemons/pacemakerd/Makefile.am]: And dropped.

**Pull request merged**